### PR TITLE
Xcm chains update

### PR DESCRIPTION
**Pull request by DRadmir**

### CODE REVIEW

File: xcm/xcm_token_locations.json [hunk 1]
```diff
@@ -243,22 +463,218 @@
...
+              "parachain": 2090
+            },
+            {
+              "generalIndex": 0
+            }
+          ]
+        }
+      },
+      {
+        "symbol": "RMRK",
```
Comment (VitalyPeryatin) [inline xcm/xcm_token_locations.json:483]: `RMRK` is a token in Kusama ecosystem. It must have `v3` version for `interiors` block

File: xcm/xcm_token_locations.json [hunk 2]
```diff
@@ -243,22 +463,218 @@
...
+              "parachain": 2001
+            },
+            {
+              "generalKeyV1": "0x0207"
+            }
+          ]
+        }
+      },
+      {
+        "symbol": "vKSM",
```
Comment (VitalyPeryatin) [inline xcm/xcm_token_locations.json:514]: `vKSM` is a token in Kusama ecosystem. It must have `v3` version for `interiors` block

File: xcm/xcm_token_locations.json [hunk 3]
```diff
@@ -105,40 +311,54 @@
...
+              "generalKeyV3": {
                 "length": 2,
                 "data": "0x0080000000000000000000000000000000000000000000000000000000000000"
               }
             }
           ]
         }
       },
+      {
+        "symbol": "LKSM",
```
Comment (VitalyPeryatin) [inline xcm/xcm_token_locations.json:334]: `LKSM` is a token in Kusama ecosystem. It must have `v3` version for `interiors` block

File: xcm/xcm_token_locations.json [hunk 4]
```diff
@@ -243,22 +463,218 @@
...
+          "v1": [
+            {
+              "parachain": 2105
             },
             {
-              "GeneralIndex": 0
+              "palletInstance": 5
             }
           ]
         }
```
Comment (VitalyPeryatin) [inline xcm/xcm_token_locations.json:675]: Everything is good. Let's just check again that all tokens in Kusama ecosystem have `v3` version for `interiors` block

File: xcm/xcm_token_locations.json (hunk 1 above)
Comment (SkyForger79) [reply xcm/xcm_token_locations.json:494]: Fixed

File: xcm/xcm_token_locations.json (hunk 2 above)
Comment (SkyForger79) [reply xcm/xcm_token_locations.json:547]: Fixed

File: xcm/xcm_token_locations.json (hunk 3 above)
Comment (SkyForger79) [reply xcm/xcm_token_locations.json:334]: Fixed

**Review by VitalyPeryatin — APPROVED** [review]

File: xcm/xcm_token_locations.json (hunk 4 above)
Comment (VitalyPeryatin) [reply xcm/xcm_token_locations.json:812]: Fixed

**Review by SkyForger79 — APPROVED** [review]

**Pull request merged**